plugins/faustwp/includes/updates/upgrade-database.php

```diff
@@ -7,7 +7,13 @@
 
 namespace WPE\FaustWP\Updates;
 
-add_action( 'plugins_loaded', __NAMESPACE__ . '\\upgrade_database' );
+add_action( 'plugins_loaded', __NAMESPACE__ . '\\call_upgrade_database_action' );
```

Comment:
Was this a phpstan recommendation?

Reply:
Yes it was complaining about `upgrade_database` function returning boolean when it didn't have to. We use this function for testing and we check the return value.